SwaggerGen/Swagger/Swagger.php

```diff
@@ -145,14 +145,19 @@ public function handleCommand($command, $data = null)
             case 'model!':
 			case 'definition':
             case 'definition!':
-				$definition = new Schema($this);
-				if(substr($command, -1) === '!') {
-				    $definition->setReadOnly();
-                }
 				$name = self::wordShift($data);
 				if (empty($name)) {
 					throw new \SwaggerGen\Exception('Missing definition name');
 				}
+				$typeDef = $data;
```

Comment:
Unless all of $data (including whitespace) is supposed to be eaten (i.e. a description), please use `self::wordShift($data)`, which trims whitespace (which would trip `empty()` up in this particular situation) and enables forward compatibility. Not applicable in this case, but in case `$data` **is** supposed to be eaten, it should be `trim()`'ed to chop of whitespace at the ends, unless the whitespace is significant.